src/components/search-bar/search-result-header/SearchResultHeader.tsx

```diff
@@ -1,32 +1,67 @@
 import * as React from "react";
-import { useDispatch } from "react-redux";
-import { openModal } from "../../../core/modal.slice";
+
+import {
+  FilterItemTerm,
+  TermOnClickHandler
+} from "../../../apps/search-result/types";
+import {
+  FacetResult,
+  useSearchFacetQuery
+} from "../../../core/dbc-gateway/generated/graphql";
 import { useText } from "../../../core/utils/text";
-import { FacetBrowserModalId } from "../../facet-browser/FacetBrowserModal";
-import ButtonTag from "../../Buttons/ButtonTag";
+import {
+  formatFacetTerms,
+  getPlaceHolderFacets,
+  lineFacets
+} from "../../facet-browser/helper";
+import FacetLine from "../../facet-line/FacetLine";
+import FacetLineSelected from "../../facet-line/FacetLineSelected";
 
 export interface SearchResultHeaderProps {
   hitcount: string;
   q: string;
+  filters: { [key: string]: { [key: string]: FilterItemTerm } };
+  filterHandler: TermOnClickHandler;
 }
 
 const SearchResultHeader: React.FC<SearchResultHeaderProps> = ({
   hitcount,
-  q
+  q,
+  filters,
+  filterHandler
 }) => {
   const t = useText();
-  const dispatch = useDispatch();
+
+  const { data } = useSearchFacetQuery(
+    {
+      q: { all: q },
+      facets: lineFacets,
+      facetLimit: 10,
+      filters: formatFacetTerms(filters)
+    },
+    {
+      keepPreviousData: true,
+      placeholderData: {
+        search: {
+          facets: getPlaceHolderFacets(lineFacets)
+        }
+      }
+    }
+  );
```

Comment:
This is misunderstood.

1. The facet line is based on the `intelligentFacets` query - not the standard one.
2. The query should not use a limited/different sent of facets compared to the standard one
3. It is the number of terms for each facet that determines whether the facet should be shown as a dropdown (multiple terms) or as a button (single term).